virtual_workstation_setup.sh

```diff
@@ -155,8 +155,9 @@ setup_master() {
 
             python -m pip install "pip==21.0.1"
 
-            python -m pip install -r etc/pip/dev-requirements.txt
-            python -m pip install keyring
+            python -m pip install 'poetry==1.5.1'
+            export PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring
```

Comment:
I see you're exporting a new environment variable, but it's not clear where it's used. Can you add a comment noting how/where the variable is picked up?

Reply:
done